catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/utils/FormulaEditorWrapper.java

```diff
@@ -130,6 +131,11 @@ public void performClickOn(Matcher<View> matcher) {
 				.perform(click());
 	}
 
+	public void performDoubleClickOn(Matcher<View> matcher) {
```

Comment:
this is so specific to a single test that it doesnt have any business in here. please move to the test, (but use the static matcher from the wrapper)